analytics/population/nc_pop_recent.ipynb

```diff
@@ -0,0 +1,2597 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.    census_NC= census_NC[['TRACTCE20','GEOID20', 'NAME', 'NC_ID', 'INTPTLAT20', 'INTPTLON20', 'geometry']]
</pre><p>nit: space before "=".</p><p> </p><p>So the <code>geometry</code> here is something produced by <code>overlay</code>,right? It would be good to explain that and what <code>geometry</code> represents since it's important down the road.    </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/hackforla/311-data/pull/1336/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Added the space. Added details about the geometry column. overlay function is like merge function for shape files. The merge can be union or difference and so on.</p>

Reply:
<p>Hmm I don't see the comments? Also, there is still no space right now.</p>